<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplicationSettingsView</class>
 <widget class="QDialog" name="ApplicationSettingsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Application configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabApplicationConfiguration">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabApplicationDbConfiguration">
      <attribute name="title">
       <string>Database configuration</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formApplicationDbConfiguration">
         <property name="verticalSpacing">
          <number>4</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lblAplDatabaseServer">
           <property name="text">
            <string>Database server:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lnAplDatabaseServer"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblAplDatabase">
           <property name="text">
            <string>Database:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lnAplDatabase"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblAplUserID">
           <property name="text">
            <string>UserID:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lnAplUserID"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblAplPassword">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lnAplPassword"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLoggingConfiguration">
      <attribute name="title">
       <string>Logging configuration</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLoggingConfiguration">
         <item row="0" column="0">
          <widget class="QLabel" name="lblLogLogging">
           <property name="text">
            <string>Logging Server:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lnLogLoggingServer"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblLogLoggingDatabase">
           <property name="text">
            <string>Logging Database:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lnLogLoggingDatabase"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblLogUserId">
           <property name="text">
            <string>UserId:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lnLogUserId"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblLogPassword">
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lnLogPassword"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btnConfig">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnConfig</sender>
   <signal>accepted()</signal>
   <receiver>ApplicationSettingsView</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnConfig</sender>
   <signal>rejected()</signal>
   <receiver>ApplicationSettingsView</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
